lib/led/led.js

```diff
@@ -427,7 +432,7 @@ Led.prototype.fade = function(val, time, callback) {
   }.bind(this);
 
   return this.animate({
-    duration: time || 1000,
+    duration: time,
     complete: complete,
     step: step
   });
```

Comment:
I've got a slew of tests I plan to write :-) I was just waiting on #710 to be merged first.